ydb/library/yql/public/udf/arrow/block_item.h

```diff
@@ -64,25 +67,30 @@ class TBlockItem {
 
     // TODO: deprecate AsTuple() in favor of GetElements()
     inline const TBlockItem* AsTuple() const {
-        Y_DEBUG_ABORT_UNLESS(Raw.GetMarkers() == EMarkers::Present);
+        Y_DEBUG_ABORT_UNLESS(GetMarkers() == EMarkers::Embedded);
         return Raw.Tuple.Value;
     }
 
     inline const TBlockItem* GetElements() const {
-        Y_DEBUG_ABORT_UNLESS(Raw.GetMarkers() == EMarkers::Present);
+        Y_DEBUG_ABORT_UNLESS(GetMarkers() == EMarkers::Embedded);
         return Raw.Tuple.Value;
     }
 
     inline TBlockItem GetElement(ui32 index) const {
-        Y_DEBUG_ABORT_UNLESS(Raw.GetMarkers() == EMarkers::Present);
+        Y_DEBUG_ABORT_UNLESS(GetMarkers() == EMarkers::Embedded);
         return Raw.Tuple.Value[index];
     }
 
     inline TStringRef AsStringRef() const {
-        Y_DEBUG_ABORT_UNLESS(Raw.GetMarkers() == EMarkers::Present);
+        Y_DEBUG_ABORT_UNLESS(GetMarkers() == EMarkers::String);
         return TStringRef(Raw.String.Value, Raw.String.Size);
     }
 
+    inline TStringRef GetEmbeddedStringRef() const {
```

Comment:
I don't see any constructor that can initialize such state: EMarkers::Embedded. BlockItem(TStringRef value) sets EMarkers::String